notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<p>Add these imports to the cell above to have them all in one cell </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done. Thanks!</p>